lib/game_of_life_web/templates/universe/show.html.leex

```diff
@@ -1,7 +1,13 @@
-    <div id="universe-page">
-      <h1>It's time for... the Game... of... LIFE!</h1>
+<div id="universe-page">
+  <h1>It's time for... the Game... of... LIFE!</h1>
 
-      <div class="universe">
-        <%= render_cells(@cells, @dimensions) %>
-      </div>
-    </div>
\ No newline at end of file
+  <form action="#" phx-change="update_speed">
+    <label class="form-label inline middle">Slower</label>
+    <input type="range" id="speed" class="form-control slider middle" name="speed" min="1" max="11" value="" step="1">
+    <label class="form-label inline middle">Faster</label>
+  </form>
```

Comment:
Based on [this Phoenix thread](https://elixirforum.com/t/liveview-phx-change-attribute-does-not-emit-event-on-input-text/21280) it seems like `phx-change` is only supported on `<form>` elements, which is actually... a little sad 😢